src/ResourceManager/AzureBackup/Commands.AzureBackup/AzureRM.Backup.psd1

```diff
@@ -1,9 +1,9 @@
 #
-# Module manifest for module 'AzureRM.Backup'
+# Module manifest for module 'PSGet_AzureRM.Backup'
```

Comment:
@MiYanni should we keep this change as-is and report back on the original issue that `Update-ModuleManifest` is causing this issue and we can't fix the header until we move to the mechanism that doesn't use that cmdlet?

Reply:
Yes. We will no longer be using `Update-ModuleManifest` in the future (for Az). So, the issue will persist for `AzureRM` but will not happen in `Az`. That should just be the result of the issue. We don't need to do anything for AzureRM.